SECURITY.md

```diff
@@ -0,0 +1,123 @@
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+<!-- Copyright Contributors to the Rez Project -->
+
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you think you've found a potential vulnerability in rez, please
+report it by filing a GitHub [security
+advisory](https://github.com/AcademySoftwareFoundation/rez/security/advisories/new). Alternatively,
+email security@rez-project.io and provide your contact info for further
+private/secure discussion.  If your email does not receive a prompt
+acknowledgement, your address may be blocked.
+
+Our policy is to acknowledge the receipt of vulnerability reports
+within 48 hours. Our policy is to address critical security vulnerabilities
+rapidly and post patches within 14 days if possible.
+
+## Known Vulnerabilities
+
+There are currently no known reported vulnerabilities.
```

Comment:
This is technically not true. We have an arbitrary code execution vulnerability, see https://github.com/AcademySoftwareFoundation/rez/issues/937.

Reply:
added a line